prow/cmd/hook/events.go

```diff
@@ -81,3 +88,19 @@ func (ea *EventAgent) handleIssueCommentEvent(ic github.IssueCommentEvent) {
 		}
 	}
 }
+
+func (ea *EventAgent) handleStatusEvents(se github.StatusEvent) {
+	l := logrus.WithFields(logrus.Fields{
+		"context": se.Context,
+		"sha":     se.Context,
+		"state":   se.State,
+		"id":      se.ID,
+		"repo":    se.Repo.Name,
+	})
+	l.Infof("Status description %s.", se.Description)
+	for p, h := range ea.Plugins.StatusEventHandlers(se.Repo.FullName) {
+		if err := h(ea.Plugins, se); err != nil {
```

Comment:
Pass in a `plugins.PluginClient` with the logger `l` as in the other two handlers.